docs/tutorials/open-source/apache-flink/1_single_feature.qmd

```diff
@@ -0,0 +1,284 @@
+# A real-life use case: fraud detection (part 1)
+
+Imagine you’re a data scientist who works at a large bank. You have been tasked with one of the most challenging problems in banking today: identifying fraudulent transactions. The bank receives transaction details from its credit card customers in a Kafka topic, which include information about the transaction date and time, transaction amount, transaction location, merchant, category of purchase, and so on. Given the nature of the data, you want to use Apache Flink for its stream processing capabilities and to develop machine learning features that can be used to identify fraud.
+
+
+### Prerequisites
+
+* Docker Compose: This tutorial uses Docker Compose to manage an Apache Kafka environment (including sample data generation) and a Flink cluster (for remote execution). You can [download and install Docker Compose from the official website](https://docs.docker.com/compose/install/).
+* JDK 11 release: Flink requires Java 11.
+* Python 3.9 or 3.10.
+* Follow [the instructions on the Ibis project page](https://ibis-project.org/install) to install the Flink backend for Ibis. For the tutorial below, we assume that you already have the Ibis package correctly installed in your environment.
```

Comment:
same feedback -- I would just link back to the setup part of the tutorial and assume the user has followed that for this page